res/skins/Tango/buttons/btn_beatbox_up.svg

```diff
@@ -1,2 +1,60 @@
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-<svg id="svg3063" width="16" height="20" version="1.1" xmlns="http://www.w3.org/2000/svg"><path id="path3065" d="m6.9997 5.0819-4.5715 7.9181h9.143z" fill="#888"/><path id="path3067" d="m12.5 0c1.9304 0 3.5 1.5696 3.5 3.5v13c0 1.9304-1.5696 3.5-3.5 3.5h-10c-0.9806 0-1.864-0.3796-2.5-1.0312v-2.4688c0 1.3936 1.1064 2.5 2.5 2.5h10c1.3936 0 2.5-1.1064 2.5-2.5v-13c0-1.3936-1.1064-2.5-2.5-2.5h-10c-1.3936 0-2.5 1.1064-2.5 2.5v-2.4375c0.63596-0.6516 1.5194-1.0625 2.5-1.0625h10z" color="#000000" fill="#666" style="block-progression:tb;text-indent:0;text-transform:none"/></svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Forgot to use optimized SVGs?

Reply:
Or maybe this SVG change was unintentional anyway?

Reply:
yes, that was a test because suddenly --after changing the icon paths-- only that icon would not show up. any other in taht place would work.
I'll optimizeto see if that works

Reply:
fixed.